tests/dependencies_test.go

```diff
@@ -99,10 +99,19 @@ func installWordpressBundle(p *porter.TestPorter) (namespace string) {
 	return namespace
 }
 
-func cleanupWordpressBundle(p *porter.TestPorter) {
+func cleanupWordpressBundle(p *porter.TestPorter, targetDependencyName string) {
```

Comment:
It's not clear why changing this test cleanup function was necessary? What problem were you running into that made you change it so I can help?